hapi/hedera-protobufs/services/auxiliary/tss/tss_share_signature.proto

```diff
@@ -0,0 +1,83 @@
+/**
+ * # Tss Share Signature
+ * Represents a transaction that submits a node's share signature on a block hash
+ * during the TSS (Threshold Signature Scheme) process.
+ *
+ * ### Keywords
+ * The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT",
+ * "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this
+ * document are to be interpreted as described in
+ * [RFC2119](https://www.ietf.org/rfc/rfc2119) and clarified in
+ * [RFC8174](https://www.ietf.org/rfc/rfc8174).
+ */
+syntax = "proto3";
+
+package com.hedera.hapi.services.auxiliary.tss;
+
+/*
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+option java_package = "com.hedera.hapi.services.auxiliary.tss.legacy";
+// <<<pbj.java_package = "com.hedera.hapi.services.auxiliary.tss">>> This comment is special code for setting PBJ Compiler java package
+option java_multiple_files = true;
+
+/**
+ * The body of a transaction that includes a node's signature of a block hash using its private
+ * shares within the TSS process.
+ * This transaction will be prioritized for gossip transmission.
```

Comment:
There should be a summary line here, also, note that single newlines are collapsed in markdown, so you probably _intended_ to have a `<br/>` tag after `process.` at the end of the second line.

A _possible_ summary line might be `A TSS transaction Body.<br/>`.

Alternatively, this third line looks like a requirement:
```suggestion
 * A TSS transaction Body.<br/>
 * This transaction body communicates a node's signature of a block hash
 * using its private share within the TSS process.
 *
 * This transaction MUST be prioritized for low latency gossip transmission.
```

Reply:
Fixed as per suggestion. Thanks @jsync-swirlds